src/optimizer/query_rewrite.c

```diff
@@ -4667,7 +4672,38 @@ qo_convert_to_range (PARSER_CONTEXT * parser, PT_NODE ** wherep)
 	    }
 
 	  arg1_prior = pt_get_first_arg_ignore_prior (dnf_node);
-	  if (!pt_is_attr (arg1_prior) && !pt_is_instnum (arg1_prior))
+
+	  is_attr = true;
```

Comment:
is_attr is declared as `int `type

Reply:
it is legacy code. 
In order to fix, pt_is_attr()'s return type need to be changed. but there are so many caller of it.
I think It doesn't matter if don't fix it.